api/Document.json

```diff
@@ -4632,10 +4632,34 @@
       },
       "exitPictureInPicture": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/exitPictureInPicture",
           "support": {
-            "chrome": {
-              "version_added": "69"
-            },
+            "chrome": [
+              {
+                "version_added": "69",
+                "version_removed": "72",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "enable-experimental-web-platform-features",
+                    "value_to_set": "enabled"
+                  },
+                  {
+                    "type": "preference",
+                    "name": "enable-surfaces-for-videos",
+                    "value_to_set": "enabled"
+                  },
+                  {
+                    "type": "preference",
+                    "name": "enable-picture-in-picture",
+                    "value_to_set": "enabled"
+                  }
+                ]
+              },
+              {
+                "version_added": "72"
```

Comment:
Can you sort this entry to the top? The first entry should be the entry with the broadest interest to web developers